contracts/Controller.sol

```diff
@@ -116,24 +116,36 @@ contract Controller is Initializable, OwnableUpgradeSafe, ReentrancyGuardUpgrade
     /**
      * @notice modifier check if protocol is not paused
      */
-    modifier isNotPaused {
-        require(!systemPaused, "Controller: system is paused");
-
+    modifier notPaused {
+        _isNotPaused();
         _;
     }
 
     /**
      * @notice modifier to check if sender is an account owner or an authorized account operator
-     * @param _sender sender address
      * @param _accountOwner account owner address
      */
-    modifier isAuthorized(address _sender, address _accountOwner) {
+    modifier onlyAuthorized(address _accountOwner) {
+        _isAuthorized(_accountOwner);
+        _;
+    }
+
+    /**
+     * @dev check the system is not paused
+     */
+    function _isNotPaused() internal view {
+        require(!systemPaused, "Controller: system is paused");
+    }
+
+    /**
+     * @dev check the sender is an authorized operator.
+     * @param _accountOwner owner of a vault
+     */
+    function _isAuthorized(address _accountOwner) internal view {
         require(
-            (_sender == _accountOwner) || (operators[_accountOwner][_sender]),
+            (msg.sender == _accountOwner) || (operators[_accountOwner][msg.sender]),
             "Controller: msg.sender is not authorized to run action"
         );
-
-        _;
     }
```

Comment:
Why this modification ?

Reply:
save some bytes for passing in that parameter

Reply:
Would like to keep it with two arguments, it much clearer that way, unless it is saving a lot of bytes...

Reply:
But also why having a modifier calling the function ?

Reply:
because if it's a modifier, it copy all the code in the modifier into the function body.

Reply:
So if we have multiple function with the same modifier, better use an internal function in the modifier, so the same code won't be copied multiple times

Reply:
I'll make it back to using 2 arguments then